CriticalMass/FormatDisplay.swift

```diff
@@ -45,9 +42,11 @@ enum FormatDisplay {
         return formatter.string(from: components)
     }
 
-    static func hoursAndMinutesDateString(from message: ChatMessage) -> String {
+    static func hoursAndMinutesDateString(from message: ChatMessage, calendar: Calendar = .current) -> String {
```

Comment:
Could be moved to the DateFormatter extension

Reply:
Yes, but that's maybe something for a future PR